src/System.Net.WebSockets.WebSocketProtocol/tests/System.Net.WebSockets.WebSocketProtocol.Tests.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
Shouldn't this also test some version of .NETCore since it looks like we're building a version that would apply there?  I know we wouldn't encourage it's use in new code, but I think someone who build a library using this package would continue to use the package version on .NET Core so we should test it.

Reply:
Yes. I added net6.0. It now works.

I also made sure to preserve net48, because there are some async APIs that are unavailable in net462. And this is consistent with the 5.0 source: https://github.com/dotnet/runtime/blob/4aadfea70082ae23e6c54a449268341e9429434e/src/libraries/System.Net.WebSockets.WebSocketProtocol/tests/System.Net.WebSockets.WebSocketProtocol.Tests.csproj#L3